.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "cmake-modules"]
 	path = cmake-modules
 	url = https://github.com/bilke/cmake-modules.git
+[submodule "contrib/farmhash"]
```

Comment:
I think we can move farmhash's source code into our code base, it is archived, and just two relative source